src/main/java/uk/ac/ebi/embl/api/validation/check/file/FileValidationCheck.java

```diff
@@ -211,6 +211,18 @@ protected int getAssemblyLevel(ValidationScope scope)
 		return assemblyLevel;
 	}
 
+	// TODO: Genome length check should be generic given the tax id for all species with known expected genome length.
+	public void validateCovid19GenomeSize() throws ValidationEngineException {
+		if (sharedInfo.assemblyType != null && sharedInfo.assemblyType.equalsIgnoreCase(AssemblyType.COVID_19_OUTBREAK.getValue())) {
+
+			long genomeSize = GenomeUtils.calculateGenomeSize(sharedInfo.sequenceInfo, sharedInfo.agpPlacedComponents);
+
+			if (genomeSize > GenomeUtils.COVID_19_OUTBREAK_GENOME_MAX_SIZE) {
+				throw new ValidationEngineException(String.format("%s maximum genome size is %d bp.", AssemblyType.COVID_19_OUTBREAK.getValue(), GenomeUtils.COVID_19_OUTBREAK_GENOME_MAX_SIZE), ReportErrorType.VALIDATION_ERROR);
```

Comment:
The ValidationEngineExceptions are meant to be system errors not user errors. However, what we are generating here is an user and needs to be reported as such back to the user. ValidationEngineExceptions usage is unexpected. What I would expect to see that we add a new error in ValidationResult. Possibly we could use throwValidationCheckException.